docs/localization.rst

```diff
@@ -67,25 +66,25 @@ Adding a new Locale
 
 #. Create the `jsi18n` file for the new locale::
 
-        $ ./manage.py compilejsi18n
+        make compilejsi18n
 
 #.  Verify django loads new locale without errors by visiting the locale's home
     page. E.g., https://developer-local.allizom.org/ml/
 
 #.  BONUS: Use `podebug` to test a fake translation of the locale::
 
-        $ cd locale
-        $ podebug --rewrite=bracket templates/LC_MESSAGES/django.pot ml/LC_MESSAGES/django.po
-        $ ./compile-mo.sh .
+        cd locale
+        podebug --rewrite=bracket templates/LC_MESSAGES/django.pot ml/LC_MESSAGES/django.po
+        ./compile-mo.sh .
```

Comment:
http://dennis.readthedocs.io/en/latest/translating.html

As I recall, there's a bug in some of the translation transforms where they don't take variables into account. I started fixing that, but I don't think I ever finished. :(